src/init.js

```diff
@@ -0,0 +1,176 @@
+  var timekitSetupConfig = function() {
+    sdk.configure(getConfig().sdk);
+  };
+
+  // Initilization method
+  var init = function(suppliedConfig) {
+
+    var localConfig = config.setDefaults(suppliedConfig || {});
+    config.update(localConfig);
+
+    utils.logDebug(['Supplied config:', suppliedConfig]);
+
+    try {
+
+      // Set rootTarget to the target element and clean before child nodes before continuing
+      render.prepareDOM(suppliedConfig || {});
+
+      // Start from local config
+      if (!suppliedConfig || (!suppliedConfig.project_id && !suppliedConfig.project_slug) || suppliedConfig.disable_remote_load) {
+        return startWithConfig(suppliedConfig)
+      }
+
+    } catch (e) {
+      utils.logError(e)
+      return this
+    }
+
+    // Load remote config
+    loadRemoteConfig(suppliedConfig)
+    .then(function (response) {
+      var remoteConfig = response.data
+      // streamline naming of object keys
+      if (remoteConfig.id) {
+        remoteConfig.project_id = remoteConfig.id
+        delete remoteConfig.id
+      }
+      // merge with supplied config for overwriting settings
+      var mergedConfig = $.extend(true, {}, remoteConfig, suppliedConfig);
+      utils.logDebug(['Remote config:', remoteConfig]);
+      startWithConfig(mergedConfig)
+    })
+    .catch(function (e) {
+      render.triggerError(['The project could not be found, please double-check your project_id/project_slug', e]);
+    })
+
+    return this
+
+  };
+
+  // Load config from remote (embed or hosted)
+  var loadRemoteConfig = function(suppliedConfig) {
+
+    var localConfig = config.setDefaults(suppliedConfig);
+    config.update(localConfig);
+    timekitSetupConfig();
+    if (suppliedConfig.project_id && suppliedConfig.app_key) {
+      return sdk
+      .makeRequest({
+        url: '/projects/embed/' + suppliedConfig.project_id,
+        method: 'get'
+      })
+    }
+    if (suppliedConfig.project_slug) {
+      return sdk
+      .makeRequest({
+        url: '/projects/hosted/' + suppliedConfig.project_slug,
+        method: 'get'
+      })
+    }
+    throw render.triggerError('No widget configuration, project_slug or project_id found');
+
+  };
+
+  // Parse the config and start rendering
+  var startWithConfig = function(suppliedConfig) {
+
+    // Handle config and defaults
+    try {
+      config.parseAndUpdate(suppliedConfig);
+      utils.logDebug(['Final config:', getConfig()]);
+      utils.logDebug(['Version:', getVersion()]);
+    } catch (e) {
+      render.triggerError(e);
+      return this
+    }
+
+    return startRender();
+
+  };
+
+  // Render method
+  var startRender = function() {
+
+    utils.doCallback('renderStarted');
+
+    // Setup Timekit SDK config
+    timekitSetupConfig();
+
+    // Initialize FullCalendar
+    render.initializeCalendar();
+
+    // Get availability through Timekit SDK
+    render.getAvailability();
+
+    // TODO Show timezone helper if enabled
```

Comment:
is this saved for another date?

Reply:
Well, because we now more natively support resources from different timezones, it doesn't really make sense to show the timezone helper. Which timezone should it show? For which resource?

In the future, we might be able to make it more intelligent, but imma just disable it for now 👍